prime-angular-frontend/src/app/core/resources/health-authority-resource.service.ts

```diff
@@ -375,6 +375,28 @@ export class HealthAuthorityResource {
       );
   }
 
+  /**
+   * @description
+   * Checks if the provided PEC is already assigned to another site under another Health Authority
+   */
+  public checkPecExistsInOtherHEalthAuthority(healthAuthCode: number, pec: string): Observable<boolean> {
+    // In order to use this method with the AsyncValidator, we're inverting the results here.
```

Comment:
I tried doing the same yesterday, I just failed with making a name that makes sense, since the error is only when a duplicate is found (204) and if I call it for example `hasNoDuplicate` and negate that it sounds kind of weird. I will see if I can come up with a better naming

Reply:
Moved to healtha authority site information form